### PR TITLE
[Doc] PR for Document Platform support for deploying YugabyteDB on ARM

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-2655

Preview: https://deploy-preview-12761--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws/#yugabytedb-anywhere-managed-configuration

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md [hunk 1]
```diff
@@ -121,14 +121,20 @@ For deployment, YugabyteDB Anywhere aims to provide you with easy access to the
...
 You have an option to provide the following:
 
 - A custom classless inter-domain routing (CIDR) block for each regional VPC. If not provided, YugabyteDB Anywhere will choose defaults, aiming to not overlap across regions.
 
-- A custom Amazon Machine Image (AMI) ID to use in each region. For a non-exhaustive list of options, see [Ubuntu 18 and Oracle 8 support](#ubuntu-18-and-oracle-8-support). If you do not provide any values, a recent [AWS Marketplace CentOS AMI](https://wiki.centos.org/Cloud/AWS) is used.<br><br>
+- A custom Amazon Machine Image (AMI) ID to use in each region. For a non-exhaustive list of options, see [Ubuntu 18 and Oracle 8 support](#ubuntu-18-and-oracle-8-support).<br> 
 
+  To use automatic provisioning to bring up a universe on [AWS Graviton](https://aws.amazon.com/ec2/graviton/), you need to pass in the Arch AMI ID of AlmaLinux or Ubuntu.<br>
+  
+  If you do not provide any values, a recent [AWS Marketplace CentOS AMI](https://wiki.centos.org/Cloud/AWS) is used.<br>
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:134]: Would clarify this is CentOS.

Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:134]: Would clarify this is CentOS.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md [hunk 2]
```diff
@@ -121,14 +121,20 @@ For deployment, YugabyteDB Anywhere aims to provide you with easy access to the
...
 
 You have an option to provide the following:
 
 - A custom classless inter-domain routing (CIDR) block for each regional VPC. If not provided, YugabyteDB Anywhere will choose defaults, aiming to not overlap across regions.
 
-- A custom Amazon Machine Image (AMI) ID to use in each region. For a non-exhaustive list of options, see [Ubuntu 18 and Oracle 8 support](#ubuntu-18-and-oracle-8-support). If you do not provide any values, a recent [AWS Marketplace CentOS AMI](https://wiki.centos.org/Cloud/AWS) is used.<br><br>
+- A custom Amazon Machine Image (AMI) ID to use in each region. For a non-exhaustive list of options, see [Ubuntu 18 and Oracle 8 support](#ubuntu-18-and-oracle-8-support).<br> 
 
+  To use automatic provisioning to bring up a universe on [AWS Graviton](https://aws.amazon.com/ec2/graviton/), you need to pass in the Arch AMI ID of AlmaLinux or Ubuntu.<br>
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:132]: We need to communicate they also need a special ARM release. That can be found on our releases page under "Linux ARM" https://docs.yugabyte.com/preview/releases/release-notes/v2.13/ for example. 

This should be made available to the platform either via an HTTP/S3 URL (I don't think we have docs for this either, but it's the flow when you go to http://portal.dev.yugabyte.com/releases and click "Import" in the top right) or by uploading directly to the platform installation.

**Comment by chidmuthu** [comment]
@sridhar-yb

**Comment by chidmuthu** [comment]
@sridhar-yb

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:134]: Where?  (CentOS is already included in the link)

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md [hunk 3]
```diff
@@ -121,15 +121,20 @@ For deployment, YugabyteDB Anywhere aims to provide you with easy access to the
...
 
 - A custom Amazon Machine Image (AMI) ID to use in each region. For a non-exhaustive list of options, see [Ubuntu 18 and Oracle 8 support](#ubuntu-18-and-oracle-8-support). If you do not provide any values, a recent [AWS Marketplace CentOS AMI](https://wiki.centos.org/Cloud/AWS) is used.<br><br>
 
-  ![New Region Modal](/images/ee/aws-setup/aws_new_region.png)
+   <br>
+  
+  ![New Region Modal](/images/ee/aws-setup/aws_new_region.png)<br><br><br>
+  
+  To use automatic provisioning to bring up a universe on [AWS Graviton](https://aws.amazon.com/ec2/graviton/), you need to pass in the Arch AMI ID of AlmaLinux or Ubuntu. Note that this requires a YugabyteDB release for Linux x86 ARM, which is available through one of the release pages (for example, https://docs.yugabyte.com/preview/releases/release-notes/v2.13/) by clicking **Downloads > Linux ARM**. YugabyteDB Anywhere enables you to import releases via an S3 or HTTP, as described in [Upgrade the YugabyteDB software](../../../manage-deployments/upgrade-software/).
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:136]: This should just say "Linux ARM". x86/ARM are completely different, both together is confusing.

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md [hunk 4]
```diff
@@ -121,15 +121,20 @@ For deployment, YugabyteDB Anywhere aims to provide you with easy access to the
...
+If you choose to use YugabyteDB Anywhere to configure, own, and manage a full cross-region deployment of Virtual Private Cloud (VPC), YugabyteDB Anywhere will generate a YugabyteDB-specific VPC in each selected region, then interconnect them, as well as the VPC in which YugabyteDB Anywhere was deployed, using [VPC peering](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-peering.html). This mode also sets up all other relevant subcomponents in all regions, such as Subnets, Security Groups, and Routing Table entries.
 
 You have an option to provide the following:
 
 - A custom classless inter-domain routing (CIDR) block for each regional VPC. If not provided, YugabyteDB Anywhere will choose defaults, aiming to not overlap across regions.
 
 - A custom Amazon Machine Image (AMI) ID to use in each region. For a non-exhaustive list of options, see [Ubuntu 18 and Oracle 8 support](#ubuntu-18-and-oracle-8-support). If you do not provide any values, a recent [AWS Marketplace CentOS AMI](https://wiki.centos.org/Cloud/AWS) is used.<br><br>
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:130]: Here we should specify "a recent x86 CentOS image is used"

**Review by chidmuthu — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:136]: Changed

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:130]: Fixed

**Pull request merged**